scripts/core/device.yml

```diff
@@ -381,6 +381,8 @@ etors:
       desc: "[$x_bool_t] Return true if the device supports enqueing commands to read and write pipes from the host."
     - name: MAX_REGISTERS_PER_WORK_GROUP
       desc: "[uint32_t] The maximum number of registers available per block."
+    - name: EXP_COMMAND_BUFFER_SUPPORT
+      desc: "[$x_exp_command_buffer_support_level_t] Level of experimental command-buffer support."
```

Comment:
I think before we discussed just using the existing extension device query which returns a list of strings, and then reporting our extension in there forn now. Then coming up with a more thought out reporting mechanism later, if we want to return version or support for individual capabilities within the optional feature.

Reply:
Yeah I think this makes the most sense in the short term so it doesn't extend the `ur_device_info_t`. Here's a straw man suggestion for the naming convention in the extension string: `ur_exp_<feature>`. I'll open a ticket about clarifying this in the contibution guide.

Reply:
I've removed the extension to `ur_device_info_t` for now and the associated enum for the support level.